front-end/src/containers/ProductsList/ProductsList.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { connect } from "react-redux";
 import ProductCard from "../../components/ProductPageComponents/ProductCard/ProductCard";
-import { fetchCategoryProductsAPI } from "../../graphQL/api";
+import { fetchCategoryProductsAPI } from "../../api";
```

Comment:
omdat ek die methods export van 'n `index.js` file kan mens die folder direct import en hoef jy nie nog die file name te add nie. Sê nou jou project groei en jy moet baie meer api methods add dan kan jy dit in verskillende files doen en net die goed wat jy will expose aan die res van jou app weer in jou `index.js` re-export.

So bv. as jou folder so lyk
```
currencies.js
categories.js
products.js,
index.js
``` 

en elkeen van die files export wat jy wil hê dit moet dan sal jou `index.js` file so iets lyk

```js
export * from './currencies';
export * from './categories';
export * from './products';
```